app/lib/hooks/useVideoConf/index.tsx

```diff
@@ -24,58 +23,44 @@ const handleErrors = (isAdmin: boolean, error: typeof availabilityErrors[keyof t
 	return showErrorAlert(i18n.t(`${error}-body`), i18n.t(`${error}-header`));
 };
 
-export const useVideoConf = (rid: string): { showInitCallActionSheet: () => Promise<void>; showCallOption: boolean } => {
-	const [showCallOption, setShowCallOption] = useState(false);
-
-	const serverVersion = useAppSelector(state => state.server.version);
-	const jitsiEnabled = useAppSelector(state => state.settings.Jitsi_Enabled);
-	const jitsiEnableTeams = useAppSelector(state => state.settings.Jitsi_Enable_Teams);
-	const jitsiEnableChannels = useAppSelector(state => state.settings.Jitsi_Enable_Channels);
+export const useVideoConf = (
+	rid: string
+): { showInitCallActionSheet: () => Promise<void>; callEnabled: boolean; disabledTooltip?: boolean } => {
 	const user = useAppSelector(state => getUserSelector(state));
+	const serverVersion = useAppSelector(state => state.server.version);
 
-	const [permission, requestPermission] = Camera.useCameraPermissions();
+	const { callEnabled, disabledTooltip } = useVideoConfCall(rid);
 
-	const isServer5OrNewer = compareServerVersion(serverVersion, 'greaterThanOrEqualTo', '5.0.0');
+	const [permission, requestPermission] = Camera.useCameraPermissions();
```

Comment:
They are already grouped by type, just because they are hooks doesn't mean they are the same.
useAppSelector one after another, hooks related to the action sheet ditto.